s3/dispatch.c

```diff
@@ -284,7 +285,11 @@ callback_response(void * cookie, struct http_response * res)
 			/* We have no Content-Length header. */
 			clen = (uint32_t)(-1);
 		} else {
-			clen = strtoull(s_clen, NULL, 0);
+			/* Any base, no trailing whitespace. */
+			if (PARSENUM_EX(&clen, s_clen, 0, 1)) {
```

Comment:
Isn't this allowing trailing whitespace?

Reply:
Wow.  I have no clue how that happened.